src/main/java/io/github/pascalklassen/pokefuture/PokemonService.java

```diff
@@ -17,22 +66,90 @@
 import org.slf4j.LoggerFactory;
 
 import java.lang.reflect.Field;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 public final class PokemonService {
 
+    private static final Map<Class<?>, Cache<String, Object>> caches;
+    private static boolean isCache = true;
```

Comment:
Nenn die Variable am besten in `cachingEnabled` um damit man direkt beim lesen erkennt, dass das Caching optional ist.